pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "admin_backend"
-version = "2.17.27"
+version = "2.17.29"
```

This will need bumping to `2.17.30`
```suggestion
version = "2.17.30"
```